editoast/src/models/stdcm_log.rs

```diff
@@ -1,12 +1,17 @@
 use chrono::DateTime;
 use chrono::Utc;
 use editoast_derive::Model;
+use editoast_models::DbConnection;
+use opentelemetry::trace::TraceContextExt;
 use serde::Deserialize;
 use serde::Serialize;
+use tracing_opentelemetry::OpenTelemetrySpanExt;
 use utoipa::ToSchema;
 
 use crate::core::stdcm::Request;
 use crate::core::stdcm::Response;
+use crate::models::prelude::*;
+use crate::views::Authentication;
```

Comment:
A `views` import in `models` :/

Given how things are going, the authentication and authorization are likely to stay in `views` and in a dedicated crate. The `models` "crate" won't probably know about that layer. Wdyt about taking an `Option<i64>` instead?

Reply:
Good catch, so agreed. I really quickly prototyped this solution (maybe I should have opened a draft). This is one thing to improve for sure.